tracing_test.go

```diff
@@ -171,6 +175,54 @@ func TestStartChild(t *testing.T) {
 	}
 	c.Check(t, span)
 	c.Check(t, child)
+
+	events := transport.Events()
+	if got := len(events); got != 1 {
+		t.Fatalf("sent %d events, want 1", got)
+	}
+	want := &Event{
+		Type:        transactionType,
+		Transaction: "Test Transaction",
+		Contexts: map[string]interface{}{
+			"trace": &TraceContext{
+				TraceID: span.TraceID,
+				SpanID:  span.SpanID,
+				Op:      span.Op,
+			},
+		},
+		Tags: nil,
+		// TODO(tracing): Set Transaction.Data here or in
+		// Contexts.Trace, or somewhere else. Currently ignored.
+		Extra: nil,
```

Comment:
Same comment as in https://github.com/getsentry/sentry-go/pull/329/ once merged.

Reply:
We're not really exercising tags and extra here, I'll just remove it.